Content.Shared/CartridgeLoader/CartridgeLoaderComponent.cs

```diff
@@ -32,8 +32,8 @@ public sealed partial class CartridgeLoaderComponent : Component
     /// <summary>
     /// The maximum amount of programs that can be installed on the cartridge loader entity
     /// </summary>
-    [DataField]
-    public int DiskSpace = 5;
+    [DataField("diskSpace")]
```

Comment:
```suggestion
    [DataField]
```
the name is automatically converted from camelCase to PascalCase

Reply:
Fixed